decidim-elections/app/forms/decidim/elections/censuses/internal_users_form.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module Censuses
+      # This class presents data for logging into the system with census data.
+      class InternalUsersForm < Decidim::Form
+        validate :user_authenticated
+
+        def voter_uid
+          @voter_uid ||= election.census.users(election).find_by(id: current_user&.id)&.to_global_id&.to_s
+        end
+
+        def election
+          @election ||= context.election
+        end
+
+        def current_user
+          @current_user ||= context.current_user
+        end
+
+        def current_organization
```

Comment:
We already have current_user, current_organization in the context. so those are not needed here. 
Also, you could use delegate for election.

Reply:
right :+1: